### DISCUSSION TITLE
QOS data can be obtained at 2.4Ghz, but QOS data frames cannot be obtained at 5Ghz

### DISCUSSION THREAD

**Discussion by 22-cod** [aircrack-ng/aircrack-ng#2638] [General]
At 5Ghz, only some administrative frames can be obtained.The QOS data frame is invisible.
Can someone help me with my problem. Thanks for any help.

**Comment by Mister-X-** [comment]
Let's start with that:
1. What version of Aircrack-ng are you using?
2. What adapter and driver are you using?
3. What distro and kernel version are you using?

**Reply by 22-cod** [reply]
thanks for Mister-X ，I have solved this problem through your other answers.
1.  I use wireshark on a linux system.
2.  I don't know this adapter,but i know this chip is mt7921au.
3.  Ubuntu 22.04.4
I use command such as ：“ sudo iw dev interface set channel 149 80MHz" and then i can see  5Ghz QOS data frames in wireshark

**Reply by 22-cod** [reply]
Hi Mister-X,I have another problem.
My USB card chipset is mt7921,and I can only use 80mhz in monitor mode.
Thus, for wifi6, 160MHz packets are difficult to obtain.Whether the mt7921 can enable 160mhz in monitor mode？
Are there any USB network cards that support 160MHz in monitor mode？
I would appreciate your help.
